### ISSUE TITLE
docs: help for `dev find-implementations-diff`

### ISSUE THREAD

**Issue by aguschin** [iterative/mlem.ai#180]
```
$ mlem dev find-implementations-diff
```

this is a command for devs - who wants to add an extension or fix something in MLEM. Currently I've added link to https://mlem.ai/doc/contributing/core which is shown if you call `--help`, but that page in fact doesn't have anything about it. It would be useful if we add some guide for devs on using this command in https://mlem.ai/doc/contributing/core or in command-reference.

**Comment by jorgeorpinel** [comment]
Hi. I see this is not in https://mlem.ai/doc/command-reference but also not being added by https://github.com/iterative/mlem.ai/pull/172/files . Is it a hidden feature?

**Comment by aguschin** [comment]
@jorgeorpinel, yes, it is hidden CLI command. You won't see it in `mlem --help`.

**Comment by jorgeorpinel** [comment]
Then just mentioning in https://mlem.ai/doc/contributing should be enough then. Or what usage info specifically do you think would be useful?

**Comment by jorgeorpinel** [comment]
p.s. please reprioritize if needed

**Comment by jorgeorpinel** [comment]
p.s. please reprioritize if needed. In my experience it's just good to have a sense of priorities in most tickets for later planning and grooming.

**Comment by aguschin** [comment]
> Then just mentioning in https://mlem.ai/doc/contributing should be enough then. Or what usage info specifically do you think would be useful?

Agreed, it should be enough.

**Comment by mike0sv** [comment]
It's old and hidden, I suggest we don't do a page for it. In https://github.com/iterative/mlem/pull/408 you can skip check for hidden commands
